packages/@lwc/ssr-compiler/src/compile-template/lwc-dynamic.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2024, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+
+import { builders as b } from 'estree-toolkit';
+import { ElementDirective, IsDirective } from '@lwc/template-compiler';
+import { expressionIrToEs } from './expression';
+
+import { ImportedComponent } from './component';
+import type { Component as IrComponent } from '@lwc/template-compiler';
+import type { Transformer } from './types';
+
+export function isLwcIsDirective(directive: ElementDirective): directive is IsDirective {
+    return directive.name === 'Is';
+}
+export const LwcDynamic: Transformer<IrComponent> = function LwcDynamic(node, cxt) {
+    const dynamicDirective = node.directives.find(isLwcIsDirective);
```

Comment:
If there is more than one `lwc:is` attribute, this will just return the first and ignore any that follow. Is this the same behavior as in `engine-core`?

Reply:
Parser throws `LWC1058: Invalid HTML syntax: duplicate-attribute. For more information` in both cases.